tests/kernel/arm_irq_vector_table/README.txt

```diff
@@ -3,7 +3,7 @@ Title: Installation of ISRs Directly in the Vector Table (ARM Only)
 Description:
 
 Verify a project can install ISRs directly in the vector table. Only for
-ARM Cortex-M3/4 targets.
+ARM Cortex-M3/4/33 targets.
```

Comment:
While you're at it, can you add M7 to this list too?

Reply:
Will do, thanks.

Reply:
Fixed.